src/components/app.jsx

```diff
@@ -38,14 +38,14 @@ class App extends React.Component {
     }
 
     getForecastCards(allForecastedDataArray) {
-        let forecastCards = [];
+        const forecastCards = [];
         allForecastedDataArray.forEach((singleForecast) => { forecastCards.push({
                 dateTime: this.convertToLocaleDateTime(singleForecast.dt),
                 weather: singleForecast.weather[0].description,
-                weatherIconId: singleForecast.weather[0].id,
+                weatherIconUrl: `http://openweathermap.org/img/w/${singleForecast.weather[0].id}.png`,
                 temperature: singleForecast.main.temp,
```

Comment:
`temperature: -2.68` as far as I can see on the screenshot provided in App description, the format of the temperature is different